sdks/python/container/license_scripts/dep_urls_py.yaml

```diff
@@ -48,7 +48,9 @@ pip_dependencies:
   cython:
     license: "https://raw.githubusercontent.com/cython/cython/master/LICENSE.txt"
   enum34:
-    license: "https://bitbucket.org/stoneleaf/enum34/raw/c208549a93b71b948ff7bbdfd29dce8f85527916/enum/LICENSE"
+    # The original repo is down. This license taken from somebody's clone:
+    # https://github.com/jamespharaoh/python-enum34/blob/master/enum/LICENSE
+    license: "file:///tmp/license_scripts/manual_licenses/enum34/LICENSE"
```

Comment:
did you `git add` the LICENSE file? Also, it would be great if you send a PR for master with these commit(s). Thanks!

Reply:
Oops, added now, thanks! Will send a separate PR to master. Do you prefer if I remove it from here?

Reply:
Feel free to keep it here as well, we can merge these into 2.24.0 as two separate commits. Thanks!

Reply:
https://github.com/apache/beam/pull/12624